components/centraldashboard/build_image.sh

```diff
@@ -18,6 +21,9 @@ done
 docker build -f ${DOCKERFILE} -t ${IMAGE}:${TAG} .
 
 gcloud auth activate-service-account --key-file=${GOOGLE_APPLICATION_CREDENTIALS}
-gcloud docker -- push "${IMAGE}:${TAG}"
-docker tag "${IMAGE}:${TAG}" "${IMAGE}:latest"
-gcloud docker -- push "${IMAGE}:latest"
\ No newline at end of file
+
```

Comment:
Don't you want to replace docker build with use of GCB? It seems like the problem is the docker build isn't working so we need to switch to using GCB to build.

Here are some instructions
https://cloud.google.com/cloud-build/docs/running-builds/start-build-manually
Should be something like
gcloud builds submit --tag gcr.io/[PROJECT_ID]/[IMAGE_NAME] .

Reply:
The previous build_image.sh didn't have a project ID to push to and was failing as a result.